frontend/src/components/App/Layout.tsx

```diff
@@ -14,7 +14,7 @@ const useStyle = makeStyles(theme => ({
   content: {
     flexGrow: 1,
     marginLeft: 'initial',
-    [theme.breakpoints.between('sm', 'md')]: {
+    [theme.breakpoints.between('xs', 'sm')]: {
```

Comment:
@joaquimrocha do we need this margin on any screen?? cause having a blank space on mobile screen beside sidebar is still not a good UX

Reply:
Good cathc. Thanks.